src/packages/board/board.py

```diff
@@ -51,6 +59,9 @@ def validate_config(data: typing.Dict):  # pylint: disable=unused-argument
     if data['board']['cycle_at'] <= 0:
         raise ValueError("constraint cycle_at must be non-zero and positive")
```

Comment:
I'm assuming we are not explicitly adding a logging call here because this is a fatal error?

Reply:
AFAIK validators arn't  responsible for emitting the log call, because exceptions here are fatal and get logged anyways upstream.

Reply:
Yep, thats fine. Just making sure this is intended.